mrmustard/math/backend_numpy.py

```diff
@@ -440,6 +441,27 @@ def hermite_renormalized(
 
         return G
 
+    def hermite_renormalized_batch(
+        self, A: np.array, B: np.array, C: np.array, shape: Tuple[int]
+    ) -> np.array:
+        r"""Renormalized multidimensional Hermite polynomial given by the "exponential" Taylor
```

Comment:
Remove docstring here. Only the backend manager has docs, and the same docs are valid for both backends